examples/customer_service_bot/.env.example

```diff
@@ -0,0 +1,2 @@
+TG_BOT_TOKEN=bot_token
```

Comment:
I suggest we rename all the .env.example files to .env, because creating their own .env files can be too complicated for users

Reply:
Yes, I agree that it would be easier to ask users to edit `.env` files instead of creating their own.